doc/report.md

```diff
@@ -13,59 +13,126 @@ delle mutazioni sia dei conigli che dell'ambiente circostante.
 La proposta si ispira al seguente gioco: https://phet.colorado.edu/en/simulation/natural-selection.
 
 ## Processo di sviluppo adottato
-//Parlare di scrum, come lo abbiamo adottato e come abbiamo gestito il lavoro (Sprint di 2 settimane) e parlare del 
```

Comment:
Aggiungerei questa breve introduzione:
" La metologia del processo di sviluppo adottata dal gruppo durante l'arco di tempo impiegato nella realizzazione del progetto è stata **Scrum**. Come suggerito durante il corso questo approccio ha consentito di sperimentare un framework di sviluppo del software incrementale e agile, permettendo così di ottenere alla fine di ogni sprint, diviso in slot di due settimane, un prodotto finito e sempre funzionante da presentare al committente, impersonato da uno dei membri del gruppo.
Inoltre, rispettando l'organizzazione del team di **Scrum**, è stato eletto un product owner (Marta Spadoni), lo SCRUM master( _non ricordo se lo avevamo deciso_ ) e infine il team di sviluppo, rappresentato dal gruppo stesso. "

Reply:
Mi sembrava che alla fine non avessimo assegnato questi ruoli perciò non li ho inseriti

Reply:
Esatto, li avevamo citati solo all'inizio ma poi non li abbiamo assegnati. L'ho segnalato nel commento perché ho visto che in quasi tutte le relazioni queste assegnazioni le hanno fatte. Ma non è un problema credo se non le definiamo.